src/formatting_stack/linters/eastwood.clj

```diff
@@ -12,36 +12,22 @@
    (java.io File)))
 
 (def default-eastwood-options
-  ;; Avoid false positives or undesired checks:
-  (let [linters (remove #{:suspicious-test :unused-ret-vals :constant-test :wrong-tag}
-                        eastwood.lint/default-linters)]
-    (-> eastwood.lint/default-opts
-        (assoc :linters linters
-               :rethrow-exceptions? true))))
-
-(def parallelize-linters? (System/getProperty "formatting-stack.eastwood.parallelize-linters"))
-
-(def config-filename "formatting_stack.clj")
-
-(assert (io/resource (str (io/file "eastwood" "config" config-filename)))
-        "The formatting-stack config file must exist and be prefixed by `eastwood/config`
-(note that this prefix must not be passed to Eastwood itself).")
+  (-> eastwood.lint/default-opts
+      (assoc :rethrow-exceptions? true)))
 
 (defn lint! [{:keys [options]} filenames]
   (let [namespaces (->> filenames
                         (remove #(str/ends-with? % ".edn"))
                         (keep ns-name-from-filename))
         reports    (atom nil)
-        exceptions (atom nil)
-        output     (with-out-str
-                     (binding [*warn-on-reflection* true]
-                       (try
-                         (cond-> options
-                           true                 (assoc :namespaces namespaces)
-                           parallelize-linters? (update :builtin-config-files conj config-filename)
-                           true                 (eastwood.lint/eastwood (impl/->TrackingReporter reports)))
-                         (catch Exception e
-                           (swap! exceptions conj e)))))]
+        exceptions (atom nil)]
+    (with-out-str
```

Comment:
`formatting-stack.util/silence` might be helpful here, since you're not capturing the output:
	
https://github.com/nedap/formatting-stack/blob/6ca2d6ba6734bf6ecf1cdf5bb7b42adfa213b73d/src/formatting_stack/util.clj#L154